content/v1.10-docs/installation/supported-releases.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Supported Releases
+description: Supported releases, Kubernetes versions, OpenShift versions and upcoming release timeline
+---
+
+{/*
+Inspired by https://istio.io/latest/about/supported-releases/
```

Comment:
suggestion: supported releases is another thing we don't copy to versioned docs (it probably shouldn't be under "installation" in the first place), so this file can be deleted and removed from the manifest!